tests/test_fast_sincos.c

```diff
@@ -0,0 +1,82 @@
+#include <1chipml.h>
+
+static void getFastSinError(int lowerBound, int upperBound, int step, double multFactor, int degree);
+static void getFastCosError(int lowerBound, int upperBound, int step, double multFactor, int degree);
+static void getGenericError(int lowerBound, int upperBound, int step, double multFactor, int degree, 
+char* title, double(*actualFunc)(double), fast_sincos_real(*approxFunc)(fast_sincos_real, int));
+
+int main() {
+
+    int lowerBound = -100;
+    int upperBound = 100;
+    int step = 1;
+    fast_sincos_real multFactor = 0.1;
+
+    printf("\n");
+    getFastSinError(lowerBound, upperBound, step, multFactor, 1);
+    getFastSinError(lowerBound, upperBound, step, multFactor, 2);
+    getFastSinError(lowerBound, upperBound, step, multFactor, 3);
+    getFastSinError(lowerBound, upperBound, step, multFactor, 5);
+    getFastSinError(lowerBound, upperBound, step, multFactor, 7);
+    printf("\n");
+    getFastCosError(lowerBound, upperBound, step, multFactor, 1);
+    getFastCosError(lowerBound, upperBound, step, multFactor, 2);
+    getFastCosError(lowerBound, upperBound, step, multFactor, 3);
+    getFastCosError(lowerBound, upperBound, step, multFactor, 5);
+    getFastCosError(lowerBound, upperBound, step, multFactor, 7);
+
+    return 0;
```

Comment:
Should we have a proper test here? Because as of right now if I decide to break your code nobody will know since this test will always return 0.